stack.h

```diff
@@ -42,13 +42,15 @@ class runnable;
 
 extern bool indebugger;
 
+typedef std::pair<string, string> importIndex_t;
```

Comment:
Are we confident this use of std::pair won't produce problems with the gc? It would be pretty easy to make importIndex_t a struct that extends gc so we don't have to worry about this.

Reply:
We have implemented and switched to mem::pair just in case.